kogito-bom/pom.xml

```diff
@@ -53,2739 +53,1251 @@
         <version>${project.version}</version>
         <classifier>sources</classifier>
       </dependency>
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-quarkus-events-process-deployment</artifactId>
+        <artifactId>kogito-serverless-workflow-utils</artifactId>
         <version>${project.version}</version>
       </dependency>
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-quarkus-events-process-deployment</artifactId>
+        <artifactId>kogito-serverless-workflow-utils</artifactId>
         <version>${project.version}</version>
         <classifier>sources</classifier>
       </dependency>
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-springboot-events-process-kafka</artifactId>
+        <artifactId>kogito-serverless-workflow-runtime</artifactId>
         <version>${project.version}</version>
       </dependency>
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-springboot-events-process-kafka</artifactId>
+        <artifactId>kogito-serverless-workflow-runtime</artifactId>
         <version>${project.version}</version>
         <classifier>sources</classifier>
       </dependency>
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-events-mongodb</artifactId>
+        <artifactId>kogito-quarkus-common</artifactId>
         <version>${project.version}</version>
       </dependency>
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-events-mongodb</artifactId>
+        <artifactId>kogito-quarkus-common</artifactId>
         <version>${project.version}</version>
         <classifier>sources</classifier>
       </dependency>
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-quarkus-events-mongodb</artifactId>
+        <artifactId>kogito-quarkus-common-deployment</artifactId>
         <version>${project.version}</version>
       </dependency>
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-quarkus-events-mongodb</artifactId>
+        <artifactId>kogito-quarkus-common-deployment</artifactId>
         <version>${project.version}</version>
         <classifier>sources</classifier>
       </dependency>
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-quarkus-events-mongodb-deployment</artifactId>
-        <version>${project.version}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-addons-quarkus-events-mongodb-deployment</artifactId>
+        <artifactId>kogito-quarkus-workflow-common</artifactId>
         <version>${project.version}</version>
         <classifier>sources</classifier>
```

Comment:
To keep coherence, sources could be added below, wdyt?

Reply:
yeah, for this case the sources are defined before, I would say it should be changed the order in upstream